app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/hawkular_client_mixin.rb

```diff
@@ -25,7 +25,13 @@ def hawkular_options
      :verify_ssl => @ext_management_system.verify_ssl_mode}
   end
 
-  def status
-    @client.http_get('/status')
+  def verify_credentials
```

Comment:
@yaacov we had prefixes here (`hawkular_`) because when you include the mixin you don't want collisions with other possible generic methods (in this case `verify_credentials`).